tests/dialects/test_dialect.py

```diff
@@ -437,13 +437,13 @@ def test_time(self):
             },
         )
         self.validate_all(
-            "DATE_ADD(x, y, 'day')",
+            "DATE_ADD(x, y, day)",
             read={
-                "tsql": "DATEADD('day', y, x)",
+                "tsql": "DATEADD(day, y, x)",
             },
             write={
                 "postgres": UnsupportedError,
-                "tsql": "DATEADD('day', y, x)",
+                "tsql": "DATEADD(day, y, x)",
```

Comment:
Why were these tests changed?

Reply:
They seemed like TSQL tests where the parameters are not quoted. 
SELECT DATEDIFF(year, '2017/08/25', '2011/08/25') AS DateDiff;

Reply:
Let's move it to test_tsql then (or even better, let's try to check tsql in test_time's "DATE_ADD(x, 1, 'day')" case).

Reply:
Gave it a go, let me know if you think this suffices